ci/pipelines/geode-build/jinja.template.yml

```diff
@@ -42,20 +42,14 @@ SERVICE_ACCOUNT: ((!concourse-gcp-account))
 groups:
 - name: main
   jobs:
-  - Build
+  - {{ build_test.name }}
```

Comment:
Why is this called `build_test`?

Reply:
As a variable, it was chosen since we are trying to unify the `build` task with the `test` task's style, and *are* doing some testing within the `Build` job.  Perhaps `build_and_unit_test` would be better?

Initially, it was the iteration towards something that mirrored the `tests` specified below here.  The initial hope had been to add it to the `tests` list.  But as the scope and complexity of GEODE-5621 grew and we split this into the subtasks of GEODE-5657 *58 *59, it would be premature to add it to the `tests` list that is also consumed by the pull request pipeline.

So: it could perhaps be named better.  And I would somewhat expect it to be removed in any case in GEODE-5658's adjustment of the PR pipeline or in GEODE-5659's cleanup of the surrounding files.

Reply:
The names of the tasks closely align with the name of the gradle target. There is no `build_test` target.

Reply:
That's a good point.  Although, as a pedant, I'd point out that dependencies cause `./gradlew build` is equivalent to `./gradlew build test`.

But I'll go rename it to just `build`, shall I?